OutOfSchool/OutOfSchool.DataAccess/Repository/ChatMessageRepository.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using OutOfSchool.Services.Models;
+using OutOfSchool.Services.Models.ChatWorkshop;
+
+namespace OutOfSchool.Services.Repository;
+
+public class ChatMessageRepository : SensitiveEntityRepository<ChatMessageWorkshop>, IChatMessageRepository
+{
+    private readonly OutOfSchoolDbContext db;
+
+    public ChatMessageRepository(OutOfSchoolDbContext dbContext)
+        : base(dbContext)
+    {
+        db = dbContext;
+    }
+
+    public async Task<int> CountUnreadMessagesAsync(Guid workshopId)
```

Comment:
(Code style) It looks like `async`/`await` here are excessive and could be omitted. Though I'm not insisting on removing them, I suggest to use principle that less code is better.

Reply:
I removed them. Thank you!